package.json

```
@@ -5,8 +5,6 @@
   "name": "perishable-network",
   "version": "0.2.6",
   "description": "Shipping Perishable Goods Business Network",
-  "networkImage": "https://hyperledger.github.io/composer-sample-networks/packages/perishable-network/networkimage.svg",
-  "networkImageanimated": "https://hyperledger.github.io/composer-sample-networks/packages/perishable-network/networkimageanimated.svg",
   "scripts": {
```

THis is a test commit ?
```suggestion
  "scripts": {
```

---

done